model/src/test/test_model_hospitalizations.py

```diff
@@ -367,7 +367,7 @@ def test_model_hosp_with_obs_model_weekday_phosp():
         infection_to_admission_interval=inf_hosp,
         infections_varname="infections",
         weekday_effect_dist=weekday,
-        p_hosp_dist=p_hosp,
+        p_report_dist=p_hosp,
```

Comment:
I'd rename the input variable as well as the argument name, for maximum clarity.

Reply:
Addressed in 134f786